cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -28,12 +28,15 @@ import (
 	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
 	clientcmdapi "k8s.io/kubernetes/pkg/client/unversioned/clientcmd/api"
 	certutil "k8s.io/kubernetes/pkg/util/cert"
+	"k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/authentication/user"
```

Comment:
@andrewrynhard Yes, let's keep that one constant in this kubeadm package for now